OpenSim/Sandbox/ExampleHopperDevice/exampleHopperDevice.cpp

```diff
@@ -168,12 +168,28 @@ void addDeviceConsoleReporterToModel(Model& model, Device& device,
 } // namespace OpenSim
 
 
+void executeExample();
```

Comment:
I also arbitrarily chose `executeExample()` and I am happy to change it while I am revising. Any preferences: `runHopperExample` or `hopToIt()` ?